pkgs/desktops/gnome/extensions/extensionOverrides.nix

```
@@ -46,10 +47,10 @@ super: lib.trivial.pipe super [
   }))
 
   (patchExtension "ddterm@amezin.github.com" (old: {
-    nativeBuildInputs = [ gobject-introspection wrapGAppsHook ];
-    buildInputs = [ vte ];
+    nativeBuildInputs = [ gjs gobject-introspection wrapGAppsHook ];
+    buildInputs = [ vte libhandy ];
```

```suggestion
    nativeBuildInputs = [ gobject-introspection wrapGAppsHook ];
    buildInputs = [ gjs vte libhandy ];
```

Imagine you're cross-compiling from x86_64-linux to aarch64-linux, you'll certainly want `gjs` to be native to aarch64-linux so that it runs on the host. So `gjs` should be in `buildInputs`.